pkg/document/template.go

```diff
@@ -208,12 +208,33 @@ func getValuesTableTemplates() string {
 	valuesSectionBuilder.WriteString(`{{ define "chart.valuesHeader" }}## Values{{ end }}`)
 
 	valuesSectionBuilder.WriteString(`{{ define "chart.valuesTable" }}`)
+	valuesSectionBuilder.WriteString("{{ if .Sections.Sections }}")
+	valuesSectionBuilder.WriteString("{{ range .Sections.Sections }}")
+	valuesSectionBuilder.WriteString("\n")
+	valuesSectionBuilder.WriteString("\n### {{ .SectionName }}\n")
```

Comment:
Would it be possible to add another `\n` here to ensure compatibility with markdownlint rule `MD022/blanks-around-headings/blanks-around-headers`?